rhizome/host/lib/vm_setup.rb

```diff
@@ -347,17 +347,26 @@ def cloudinit(unix_user, public_key, nics)
     end.join("\n")
 
     raparams = nics.map { "ra-param=#{_1.tap}" }.join("\n")
-
+    interfaces = nics.map { "interface=#{_1.tap}" }.join("\n")
+    dnsmasq_address_ip6 = NetAddr::IPv6Net.parse(nics.first.net6).nth(3)
```

Comment:
Is it more customary to put this at the customary gateway address as well, e.g. ::1? To avoid too many "odd" addresses in the customer's prefix.

Reply:
Differently, is another viable option to pick a ULA prefix of some sort (or even get a public one from RIPE/ARIN) and put the DNS server in there? Since we control the route tables anyway, all we really would like is an address that does not conflict, right?

There are some ramifications on how the user experiences split horizon DNS with multiple interfaces, so this isn't a prescription, more like, do we have to think about that or is it foreclosed by other restrictions?